spring-cloud-gcp-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -7,6 +7,4 @@ org.springframework.cloud.gcp.autoconfigure.storage.GcpStorageAutoConfiguration,
 org.springframework.cloud.gcp.autoconfigure.trace.StackdriverTraceAutoConfiguration
 
 org.springframework.cloud.bootstrap.BootstrapConfiguration=\
-org.springframework.cloud.gcp.autoconfigure.config.GcpConfigAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.core.GcpContextAutoConfiguration,\
-org.springframework.cloud.gcp.autoconfigure.config.GoogleConfigPropertySourceLocator
```

Comment:
Why did we ever even have `GoogleConfigPropertySourceLocator` here, since it's not a Configuration class?

Reply:
I'm not sure this works.
I spent a lot of time debugging an issue due to the lack of `GoogleConfigPropertySourceLocator` here.
Meant to take this for a spin, will do it now.